frontend/apps/app/features/sessions/components/SessionForm/GitHubSessionFormPresenter/GitHubSessionFormPresenter.tsx

```diff
@@ -66,10 +109,26 @@ export const GitHubSessionFormPresenter: FC<Props> = ({
   return (
     <ArrowTooltipProvider>
       <div
-        className={`${styles.container} ${isPending ? styles.pending : ''} ${hasError ? styles.error : ''}`}
+        className={`${styles.container} ${isPending ? styles.pending : ''} ${hasError ? styles.error : ''} ${dragActive ? styles.dragActive : ''}`}
```

Comment:
[nits]
Using **clsx** might make it a bit more readable.

```suggestion
        className={clsx(styles.container, isPending && styles.pending, hasError && styles.error, dragActive && styles.dragActive)}
```

Reply:
I fixed it by https://github.com/liam-hq/liam/pull/2180/commits/df9e294e1ec28e7dd8a214987d044cf5d4f58075